doc/source/ray-core/ray-dashboard.rst

```diff
@@ -6,74 +6,280 @@ Ray provides a web-based dashboard for monitoring and debugging Ray applications
+You can access the dashboard through a URL printed when Ray is initialized (the default URL is **http://localhost:8265**) or via the context object returned from `ray.init`.
 
-If you prefer to explicitly set the port on which the dashboard will run, you can pass
-the ``--dashboard-port`` argument with ``ray start`` in the command line, or you can pass the
-keyword argument ``dashboard_port`` in your call to ``ray.init()``.
+.. tabbed:: Python SDK
+
+    .. code-block:: python
+
+        context = ray.init()
+        print(context.dashboard_url)
 
 .. code-block:: text
 
   INFO worker.py:1487 -- Connected to Ray cluster. View the dashboard at 127.0.0.1:8265.
 
 The dashboard is also available :ref:`when using the cluster launcher <monitor-cluster>`.
 
-Views
------
+.. note:: 
 
-There are 4 different views that the Dashboard provides: :ref:`Node view <dash-node-view>`,
-:ref:`Jobs view <dash-jobs-view>`, :ref:`Actors view <dash-actors-view>`, and :ref:`Logs view <dash-logs-view>`.
+  When using the Ray dashboard, it is highly recommended to also set up Prometheus and Grafana. 
+  Ray dashboard has a tight integration with them, and core features (e.g., :ref:`Metrics View <dash-metrics-view>`) wouldn't work without them.
 
-.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard/overview.png
-    :align: center
-    :width: 20%
+See :ref:`Ray Metrics <ray-metrics>` to learn how to set up Prometheus and Grafana.
 
-.. _dash-node-view:
+If you prefer to explicitly set the port on which the dashboard will run, you can pass
+the ``--dashboard-port`` argument with ``ray start`` in the command line, or you can pass the
+keyword argument ``dashboard_port`` in your call to ``ray.init()``.
 
-Node View
-~~~~~~~~~
+How to Guide
+------------
 
-The Node view lets you see resource utilization information on a per-node and per-worker basis.
-This also shows the assignment of GPU resources to specific actors or tasks.
+.. _dash-workflow-critial-system-metrics:
 
-In addition, the machine view lets you see **logs** for a node or a worker.
+See the system metrics
+~~~~~~~~~~~~~~~~~~~~~~
+
+Ray exports metrics by default. Metrics are available from the :ref:`Metrics View <dash-metrics-view>`. Here are some example metrics that are available.
+
+- The tasks, actors, and placement group broken down by states.
+- The :ref:`logical resource usage <logical-resources>` across nodes.
+- The hardware resource usage across nodes.
+- The autoscaler status.
+
+See :ref:`System Metrics Page <system-metrics>` for available metrics.
+
+.. _dash-workflow-resource-utilization:
+
+Comparing the hardware and logical resource utilization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Ray requires users to specify the amount of resources their tasks and actors will use through arguments such as ``num_cpus``, ``num_gpus``, ``memory``, and ``resource``. 
```

Comment:
We should add a link to https://docs.ray.io/en/latest/ray-core/scheduling/resources.html somewhere